### PR TITLE
Serenity November

### PR DESCRIPTION
**Pull request by camillebourjade**

### CODE REVIEW

File: content/updates/2021-11/1-TA-importexport.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+pim_announcement_alt_img: "Table Attribute"
+pim_announcement_audience:
+- EE
+- GE
+---
+
+# Importing and Exporting Table Attributes
+::: meta-data type="New" features="Governance" available="Late-November" in="EE" link-to-doc="../articles/manage-multidimensional-data-in-a-table.html#importexport-your-table-data.html"
+
+Several dedicated import and export jobs for table attributes are available to adapt to your most efficient workflows. This means more autonomy for you and table attributes that become more readable and easier to enrich. Your exported table attributes will be displayed exactly the way you see them in the product edit form.
```
Comment (sarahhoffmanakeneo) [inline content/updates/2021-11/1-TA-importexport.md:12]: I don't feel like this paragraph is really telling me what is being released it's very benefits heavy. And the benefits listed at good! But tell me more about the "several dedicated import and export jobs" and by telling me what they are I'll have a better sense of how they "adapt to my most efficient workflows"

File: content/updates/2021-11/2-TA-addrows.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+pim_announcement_alt_img: "Table Attribute"
+pim_announcement_audience:
+- EE
+- GE
+---
+
+# Add New Rows from the Product Edit Form
+::: meta-data type="New" features="Governance" available="Late-October" in="EE" link-to-doc="../articles/manage-multidimensional-data-in-a-table.html"
+
+Add new rows directly from the product edit form as soon as you realize they are missing. You no longer have to go back to the Settings page. Staying in the PEF will help you stay focused on enrichment and without this context-switching, your products will be ready to be shared in no time!
```
Comment (sarahhoffmanakeneo) [inline content/updates/2021-11/2-TA-addrows.md:12]: If product edit form is capitalized in the title, it should be capitalized here too

Comment (sarahhoffmanakeneo) [inline content/updates/2021-11/2-TA-addrows.md:12]:
```suggestion
Add new row options to your tables directly from the Product Edit Form. As soon as you realize a row is missing, you no longer have to go back to the Settings page to edit your Table Attribute. Staying in the PEF will help you stay focused on enrichment and get your products will be ready to be shared in no time!
```

File: content/updates/2021-11/2-TA-addrows.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_img: ./img/TA_addrows.png
+pim_announcement_alt_img: "Table Attribute"
+pim_announcement_audience:
+- EE
+- GE
+---
+
+# Add New Rows from the Product Edit Form
```
Comment (sarahhoffmanakeneo) [inline content/updates/2021-11/2-TA-addrows.md:9]:
```suggestion
# Add New Rows to the Table Attribute from the Product Edit Form
```

File: content/updates/2021-11/3-APIendpoint.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+pim_announcement_alt_img:
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# New API endpoint to delete product models
+::: meta-data type="New" features="Productivity, automation" available="Late-November" in="GE,EE" link-to-doc="https://api.akeneo.com/api-reference.html#delete_product_models__code_"
+
+We are happy to introduce a new API endpoint DELETE /api/rest/v1/product-models/{code} to quickly and easily delete a given product model with its sub-product models and variant products. This new endpoint, which completes the CRUD operations available through our REST API, allows you to manage with more flexibility and agility product family changes in your PIM. It also eases data synchronization between your sources of data (ERP) that leverage our API and your PIM.
```
Comment (sarahhoffmanakeneo) [inline content/updates/2021-11/3-APIendpoint.md:12]:
```suggestion
Introducing a new API endpoint DELETE /api/rest/v1/product-models/{code}. Quickly and easily delete a given product model with its sub-product models and variant products. This new endpoint, which completes the CRUD operations available through our REST API, allows you to manage product family changes in your PIM with more flexibility and agility. It also eases data synchronization between your sources of data (like your ERP) that leverage our API and your PIM.
```

File: content/updates/index.json [hunk 5]
```diff
@@ -1,4 +1,9 @@
 {
+  "2021-11": {
+    "title": "November 2021 Serenity updates",
+    "description": "Importing and exporting user-friendly versions of your table attributes is now possible! Add new rows and options directly from the PEF to save time and stay focused on your product enrichment. Last but not least, the possibility to delete product models from a new API end-point.",
```
Comment (sarahhoffmanakeneo) [inline content/updates/index.json:4]:
```suggestion
    "description": "Importing and exporting user-friendly versions of your table attributes is now possible! Add new rows and options to save time and stay focused on your product enrichment. Last but not least, there is now the ability to delete product models from a new API end-point.",
```

**Review by sarahhoffmanakeneo — APPROVED** [review]

File: content/updates/2021-11/2-TA-addrows.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_img: ./img/TA_addrows.png
+pim_announcement_alt_img: "Table Attribute"
+pim_announcement_audience:
+- EE
+- GE
+---
+
+# Add New Rows to the Table Attribute from the Product Edit Form
```
Comment (camillebourjade) [inline content/updates/2021-11/2-TA-addrows.md:9]:
```suggestion
# Add New Rows to the Table Attribute from the PEF
```

**Pull request merged**